runtime/src/chplsys.c

```diff
@@ -201,6 +201,13 @@ static void getCpuInfo(int* p_numPhysCpus, int* p_numLogCpus) {
   if ((f = fopen("/proc/cpuinfo", "r")) == NULL)
     chpl_internal_error("Cannot open /proc/cpuinfo");
 
+  //
+  // If f is NULL, we should have exited by now, but Coverity doesn't
+  // seem to be catching this (anymore), so I'm adding an assertion
+  // to try and help it out.
+  //
+  assert(f != NULL);
```

Comment:
chpl_internal_error() isn't listed explicitly in the  model file, but exit() is, and exit() is called by chpl_internal_error() which is why I'm surprised that this case is being flagged -- maybe Coverity gives up after a certain call depth, but I didn't think that was the case.  Since we use chpl_internal_error() a bunch but aren't seeing Coverity issues in other places, this seemed like the simpler way of dealing with the issue (primarily because I was scared to mess with the model file without a more compelling reason).  But if we see more of these over time, I agree that adding chpl_internal_error() to the model file would be worthwhile.